sdcm/microbenchmarking.py

```diff
@@ -231,7 +231,11 @@ def get_results(self, results_path, update_db):
 def main(args):
     mbra = MicroBenchmarkingResultsAnalyzer(email_recipients=args.email_recipients.split(","))
     results = mbra.get_results(results_path=args.results_path, update_db=args.update_db)
-    mbra.check_regression(results, html_report_path=args.report_path)
+    if results:
+        mbra.check_regression(results, html_report_path=args.report_path)
+    else:
+        logger.warning('Results for perf_fast_forward_output tests are empty')
+        sys.exit(1)
```

Comment:
`sys.exit()` raises exception, you don't want to make a user afraid again :)
replace by exit(1)

Reply:
changed. Is it looks ok?